cmd/minishift/cmd/addon/addon.go

```diff
@@ -27,6 +28,10 @@ const (
 	noAddOnMessage  = "No add-on with the name %s is installed."
 )
 
+var (
```

Comment:
I think util is the better place. Otherwise I understand your intention with the move. Just asking ;-)